Source/SwiftLintCoreMacros/RuleConfigurationMacros.swift

```diff
@@ -24,7 +15,7 @@ private enum RuleConfigurationMacroError: String, DiagnosticMessage {
     }
 
     var diagnosticID: MessageID {
-        MessageID(domain: "SwiftLint", id: "AutoApply.\(self)")
+        MessageID(domain: "SwiftLint", id: "RuleConfiguration")
```

Comment:
🤔 should this be `"\(self)"` or something to distinguish between the various error cases?

Reply:
Ups ... 😬